stream_alert_cli/manage_lambda/deploy.py

```diff
@@ -149,5 +156,14 @@ def deploy(options, config):
     if not terraform_generate(config=config):
         return
 
+    if 'athena' in processors:
+        # create the streamalerts table since terraform does not support this
+        # See: https://github.com/terraform-providers/terraform-provider-aws/issues/1486
+        alerts_bucket = '{}.streamalerts'.format(config['global']['account']['prefix'])
+
+        athena_opts = namedtuple('AthenaOptions', ['bucket', 'refresh_type'])
+        opts = athena_opts(alerts_bucket, 'add_hive_partition')
+        athena_create_table(opts, 'alerts', config)
```

Comment:
shouldn't this just happen on initialization?

Reply:
Yes & no - so the thought with putting it here was that the `create_table` command will first check if the table already exists and do nothing if it does. Doing it here vs the `init` workflow ensures the table gets created if it wasn't previously.. but now, as I'm typing this out, I'm not sure this is the best approach.

Reply:
Yeah, it seems a little overkill to do this on every deploy